File: Java/1610.java
import java.util.Scanner;

public class Main {
    static String mysubstr(String str, int start, int count) {
        String res = "";

        for(int i = start; i < start + count; i++){
            res += str.charAt(i);
        }

        return res;
    }

    /* 메인 함수 */
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // 입력 및 초기화
        String str = scanner.next();
        int start = scanner.nextInt();
        int count = scanner.nextInt();

        // 출력부
        System.out.println(
                Main.mysubstr(str, start, count)
        );
    }
}

File: C/1610.c
#include <stdio.h>
#include <stdlib.h>

char* mysubstr(char *str, int start, int count);

int main(void)
{
	// 입력 및 초기화 
	char* str = (char*)malloc(101);
	int start, count;
	
	gets(str);
	scanf("%d %d", &start, &count);
	
	// 계산부 
	char* res = mysubstr(str, start, count);
	
	// 출력부
	printf("%s", res);
	 
	return 0;
}

char* mysubstr(char *str, int start, int count)
{
	char* res = (char*)malloc(count);
	
	for(int i = 0; i < count; i++)
	{
		*(res + i) = *(str + start + i);
	}
	
	return res;
}
